tests/actions/ReportTest.js

```diff
@@ -35,7 +37,11 @@ describe('actions/Report', () => {
         });
     });
 
-    beforeEach(() => Onyx.clear().then(waitForPromisesToResolve));
+    beforeEach(() => {
+        const promise = Onyx.clear().then(jest.useRealTimers);
+        waitForPromisesToResolve();
```

Comment:
Should that be .then(wairForPromisesToResolve)?

Reply:
Actually the purpose is to flush timers if we use fake timers. 
I change it so it's hopefully more clear now